.github/workflows/dpf-version-test.yml

```
@@ -0,0 +1,440 @@
+    
+    - name: Install dependencies
+      run: |
+        pip install pyyaml
+        
+        # Install yq
+        wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /tmp/yq
+        chmod +x /tmp/yq
+        sudo mv /tmp/yq /usr/local/bin/yq
+    
+    - name: Update manifest versions
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2097:warning:26:1: This assignment is only seen by the forked process

(shellcheck)